src/azure/Templates/AzureLongRunningMethodTemplate.cshtml

```diff
@@ -109,6 +114,7 @@ else
 @EmptyLine
         return response
 @EmptyLine
+    @(trace_decorator)
```

Comment:
I would not do a variable, and do directly the "if Model.AddTracingDecorators" to avoid an empty line if --trace is not used

Reply:
Interestingly this does not actually create a new line.